lib/utils/new-module.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const EMBER_NAMESPACES = ['inject.controller', 'inject.service'];
+
+function buildMessage(obj) {
+  // Both the Controller module and Service module each make available the
+  // "inject" namespace. In order to make it more readable, so it's more
+  // explicit at first glance from which module the "inject" namespace
+  // belongs to or is being imported from, let's check against this so we
+  // can properly populate the import specifier to report the ESLint error.
+  // Ex: import { inject as service } from '@ember/service';
+  const isNamespace = EMBER_NAMESPACES.indexOf(`${obj.parent}.${obj.key}`) !== -1;
```

Comment:
Use `.includes()`

Reply:
This is [existing code on master](https://github.com/ember-cli/eslint-plugin-ember/blob/v6.7.1/lib/rules/new-module-imports.js#L124). It is part of a function I've extracted to an util. The goal is to reuse it between two rules. With that explanation, do you still think I should implement your suggestion?